src/storage/src/hummock/sstable_store.rs

```diff
@@ -322,18 +322,19 @@ impl SstableStore {
         self.meta_cache.clear();
     }
 
-    pub async fn sstable(
+    pub async fn sstable_syncable(
```

Comment:
If we return a impl Future, could it means that we can finish this method before pin the future? For example:

```rust
pub fn sstable_syncable(&self, sst: &SstableInfo) -> Impl Future<Output=HummockResult<(TableHolder, u64)>>
```

Reply:
> If we return a impl Future, could it means that we can finish this method before pin the future? For example:
> 
> ```rust
> pub fn sstable_syncable(&self, sst: &SstableInfo) -> Impl Future<Output=HummockResult<(TableHolder, u64)>>
> ```

we still need `stats` param, which is used in the callback of `lookup_with_request_dedup`